### DISCUSSION TITLE
fancontrol seems to be commanding a fan header but cant control?

### DISCUSSION THREAD

**Discussion by clutchmaster** [Rem0o/FanControl.Releases#2170] [General]
got a z590 aorus ultra fancontrol can take control 5 of the headers but cant for the rest was messing around with the ones i cant control and noticed one of the headers i use that cant be controlled is reacting to the slider it barely goes up and down in rpm but only noticed that after hearing the fan motor whine change pitch so it seems fan control is commanding it but cant take authority wondering whys that?

![Untitled](https://github.com/Rem0o/FanControl.Releases/assets/38575266/f959a9eb-78d0-4aed-846c-d86fda1d1b2e)

**Comment by Rem0o** [comment]
There is no "authority" on fan headers. Multiple programs can try to set a value at the same time and will compete. There is no "lock" that can be applied. Only 1 program at a time must send commands, otherwise you'll get conflicting commands.

**Reply by clutchmaster** [reply]
ah ok cool i guess the ic is just ignoring the software commands then
